Apps/Src/IOState.c

```diff
@@ -0,0 +1,92 @@
+#include "common.h"
+#include "os_cfg_app.h"
+#include "IOState.h"
+#include "Tasks.h"
+#include "Pedals.h"
+#include "Ignition.h"
+#include "Dashboard.h"
+#include "StatusLeds.h"
+
+static void putIOState(void);
+
+#define IO_STATE_DLY_MS 250u 
+#define IO_STATE_HEARTBEAT_DELAY (1000u)/IO_STATE_DLY_MS
```

Comment:
```suggestion
#define IO_STATE_HEARTBEAT_DELAY (1000u)/IO_STATE_DLY_MS
```
Maybe add another macro for 1000u called something like "IO_STATE_HEARTBEAT_MS" to avoid having magic numbers

Reply:
added it